source/translators/quickstarts/start-translating-openedx.rst

```diff
@@ -53,7 +53,7 @@ Finding a string to translate
 #. Once you have joined the edx-platform project, you will be able to select the option
    "Languages" from the left hand menu under "edx-platform"
 
-  .. image:: /_images/SelectLanguages.png
+   .. image:: /_images/SelectLanguages.png
```

Comment:
FYI, you don't need to manually number the items but the numbers were being thrown off because this image was not properly indented.  If it doesn't align with the beginning of the paragraph above it.  RST thinks you are starting a new item and so it re-started the counting.